versioned_docs/version-1.2.x/integration/sdks/emberjs.mdx

```diff
@@ -0,0 +1,464 @@
+---
+sidebar_position: 10
+---
+
+import ThemedImage from "@theme/ThemedImage";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Ember.js
+
+<ThemedImage
+  alt="Ember.js"
+  width="50"
+  className="sdk-logo"
+  sources={{
+    light: useBaseUrl("img/light/emberjs.png"),
+    dark: useBaseUrl("img/javascript-icon.png"),
```

Comment:
@zinyando I know this is still a draft but these things are easy to miss in review so wanted to update it now.  
```suggestion
    dark: useBaseUrl("img/emberjs-icon.png"),
```